res/firmware.nsh

```diff
@@ -104,6 +104,12 @@ if "%2" == "bios" then
             exit %lasterror%
         endif
 
+-        # Flash with msiefiflash and exit if possible
```

Comment:
Remove `-` at start of lines.

Reply:
Lol whoops, copy/paste error. Thanks, should be fixed now